c/zos.c

```diff
@@ -749,9 +749,9 @@ static char *makeCountedString(char *name,
 {
   int allocSize = 0;
   int len = string ? strlen(string) : 0;
-  if (len > maxLength){
+  if (len > maxLength || (len == 0 && isEntity)){
```

Comment:
What does this accomplish? Currently `safAuth_internal` will return ABEND info from SAF, not it seems to silently ignore such calls.

Let the users of the SAF wrapper deal with that, they have more context.

Reply:
Yes, we can add this check in user of library but i felt it is always better to have boundary checks in library to make it robust. 
If you feel this change is not necessary then i can cancel this commit.